Server/package.json

```diff
@@ -47,6 +47,7 @@
     "mysql": "^2.18.1",
     "pg": "^8.4.1",
     "ts-node-dev": "^1.0.0-pre.62",
-    "typeorm": "^0.2.28"
+    "typeorm": "^0.2.28",
+    "yup": "^0.32.8"
```

Comment:
Please add a brief description in the PR as to why this library was added
From a quick Google search of this [library on npm](**https://www.npmjs.com/package/yup**),

"Yup is a JavaScript schema builder for value parsing and validation. Define a schema, transform a value to match, validate the shape of an existing value, or both. Yup schema are extremely expressive and allow modeling complex, interdependent validations, or value transformations."

Reply:
Its used in our frontend already for validations